schemas/publication.json

```diff
@@ -329,6 +329,14 @@
 			"validation": [
 				"min:1"
 			]
-		}
+		},
+        "dataAvailability": {
```

Comment:
The indentation contains a mix of spaces and tabs, here and elsewhere in the PR; please follow the existing conventions for each file. (We're gradually moving to autoformatting with git hooks, but that hasn't happened everywhere yet.)